scripts/variantstore/wdl/extract/hail_create_vat_inputs.py

```diff
@@ -211,7 +217,7 @@ def write_tie_out_vcf(vds, vcf_output_path):
         vds = hl.vds.read_vds(args.vds_input_path)
         local_ancestry_file = create_vat_inputs.download_ancestry_file(args.ancestry_input_path)
 
-        main(vds, local_ancestry_file, args.sites_only_output_path)
+        main(vds, local_ancestry_file, args.sites_only_output_path, dry_run_n_parts=1)
```

Comment:
I would suggest testing on 100 partitions, to get a better representation of the performance on the full data. They will all run in parallel, so it won't add much time to the test, and this is still three orders of magnitude less than the full 150k partitions, so by Konrad's estimate it should only cost on the order of a few dollars.

Reply:
awesome! Thank you~